flexmeasures_openweathermap/__init__.py

```diff
@@ -48,6 +49,10 @@
         description=f"Name of the weather station asset, defaults to '{DEFAULT_WEATHER_STATION_NAME}'",
         level="debug",
     ),
+    "MINIMAL_WEATHER_STATIONS": dict(
```

Comment:
Code looks good, but looking at the setting name and description, you misunderstood what the setting does. It limits the geographical search radius (in degree latitude as well as longitude) in which weather stations need to be to be useful for storing the forecasts.
We always use the nearest, but if that is too far away, we rather raise.

Reply:
I thought of nearest station based on these degrees. Thank you.